sys/sema/sema.c

```diff
@@ -11,169 +12,109 @@
  *
  * @file
  *
- * @author  Christian Mehlis <mehlis@inf.fu-berlin.de>
- * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
- * @author  René Kijewski <kijewski@inf.fu-berlin.de>
+ * @author  Víctor Ariño <victor.arino@zii.aero>
```

Comment:
Yes, I'm not very clear with that, sorry. Maybe we should write a guideline on when to add/remove an author from a file (I think there's currently any, right?). The criteria I followed is: 
`sema.h` I did some significant additions but most of the code remained, so I added myself. In `sema.c` the code changed completely, so there was no code left from the previous authors, neither the authors.

I didn't want to remove the copyright because this is the legal thingy... but maybe I should remove it as well. Or do you prefer that I keep all authors and all copyrights?